helm-charts/ironic/values.yaml

```diff
@@ -264,10 +264,10 @@ ironic:
     env:
       - name: MARIADB_PASSWORD
         secretKeyRefKey: password
-        secretKeyRefName: baremetal-operator-mariadb
```

Comment:
Wow, I am surprised how it was working before consider the old name was `{{ include "ironic.fullname" . }}-baremetal-operator-mariadb`

Reply:
Looks like we're missing `IRONIC_USE_MARIADB: "true"` from the ConfigMap and therefore sqllite was use as default. I think we'll need to add it back so we can default to MariaDB and optionally use sqlite if needed.

Reply:
> Wow, I am surprised how it was working before consider the old name was `{{ include "ironic.fullname" . }}-baremetal-operator-mariadb`

Before I was joining the release-name with the secretKeyRefName in the template and that was matching with the secret key being created. Now I have made all the configMap and secret-key name static